vercel/turbo
# change init script of storybook

### Description

By running `pnpm dlx sb init --skip-install` and `pnpm install --save-dev @storybook/cli` storybook doesn't get added to the vite app properly and run into `esbuild postinstall error` which fails to start storybook. Although by running  `pnpm dlx sb init` everything works and storybook  and the dependencies get installed properly.

### Testing Instructions

Going through with the process of adding storybook to turborepo should work fine which is:
1. `cd apps`
2. `pnpm create vite`
3. add this to `.npmrc` :
```
auto-install-peers=true
legacy-peer-deps=true
node-linker=hoisted
```
4. `cd workshop`
5. `pnpm dlx sb init`
